### PR TITLE
397 heatmap fixes

### PR DESCRIPTION
**Pull request by brodly**
Fixes #{issue number here}

  - [x] Up to date with `dev` branch
  - [x] Branch name follows [guidelines](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md#feature-branching)
  - [x] All PR Status checks are successful
  - [x] Peer reviewed and approved

Any questions? See the [getting started guide](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
   },
   "scripts": {
     "start": "npm run check-env && npm run dev",
-    "dev": "webpack-dev-server --config webpack.dev.js --host 0.0.0.0",
+    "dev": "webpack-dev-server --config webpack.dev.js",
```
Comment (sellnat77) [inline package.json]: Why you do dis?

Comment (sellnat77) [reply package.json]: Does the frontend still run from docker?

Comment (brodly) [reply package.json]: afaik it doesn't (or at least I am not developing with docker). is this required for docker work?

Comment (brodly) [reply package.json]: oh and btw I did it because windows doesn't resolve 0.0.0.0 and it was annoying me ;)

**Pull request merged**